src/nvim/os/unix_defs.h

```diff
@@ -19,6 +19,10 @@
 #define SPECIAL_WILDCHAR "`'{"
 
 // Unix system-dependent file names
+
+#define NVIM_CONF_DIR "$XDG_CONFIG_HOME/nvim/"
+#define NVIM_DATA_DIR "$XDG_DATA_HOME/nvim/"
```

Comment:
Are you sure the leading slashes should be here? Wouldn't this result in `$XDG_CONFIG_HOME/nvim//...` if something is appended to it?